Buildings/Resources/Scripts/Dymola/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Tower/FanSpeed/ReturnWaterTemperature/Subsequences/Validation/Enable.mos

```diff
@@ -0,0 +1,10 @@
+simulateModel("Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Tower.FanSpeed.ReturnWaterTemperature.Subsequences.Validation.Enable", method="Cvode", stopTime=3600, tolerance=1e-06, resultFile="towFanEna");
+createPlot(id=1, position={10, 10, 1000, 1200}, y={"enaTow.uMaxTowSpeSet[1]", "enaTow.uMaxTowSpeSet[2]"}, range={0.0, 3600, 0, 0.5}, grid=true, subPlot=1, colors={{28,108,200},{0,140,72}});
+createPlot(id=1, position={10, 10, 1000, 1200}, y={"enaTow.TTowSet", "enaTow.TTow"}, range={0.0, 3600, 0, 0.5}, grid=true, subPlot=2, colors={{28,108,200},{0,140,72}});
+createPlot(id=1, position={10, 10, 1000, 1200}, y={"enaTow.uTow[1]"}, range={0.0, 3600, 0, 0.5}, grid=true, subPlot=3, colors={{28,108,200}});
+createPlot(id=1, position={10, 10, 1000, 1200}, y={"enaTow.yTow"}, range={0.0, 3600, 0, 0.5}, grid=true, subPlot=4, colors={{28,108,200}});
+createPlot(id=2, position={1020, 10, 1000, 1200}, y={"disTow.uMaxTowSpeSet[1]", "disTow.uMaxTowSpeSet[2]", "disTow.fanSpeMin"}, range={0.0, 3600, 0, 0.5}, grid=true, subPlot=1, colors={{28,108,200},{0,140,72},{238,46,47}});
+createPlot(id=2, position={1020, 10, 1000, 1200}, y={"disTow.yTow"}, range={0.0, 3600, 0, 0.5}, grid=true, subPlot=2, colors={{28,108,200}});
+createPlot(id=2, position={1020, 10, 1000, 1200}, y={"disTow1.uFanSpe","disTow1.fanSpeMin"}, range={0.0, 3600, 0, 0.5}, grid=true, subPlot=3, colors={{28,108,200},{238,46,47}});
+createPlot(id=2, position={1020, 10, 1000, 1200}, y={"disTow1.TTow", "disTow1.TTowSet"}, range={0.0, 3600, 0, 0.5}, grid=true, subPlot=4, colors={{28,108,200},{0,140,72}});
+createPlot(id=2, position={1020, 10, 1000, 1200}, y={"disTow1.yTow"}, range={0.0, 3600, 0, 0.5}, grid=true, subPlot=5, colors={{28,108,200}});
```

Comment:
I didn't see the empty line in mos script. However, I did you sometimes in the CI test, it may fail because last line is empty., but don't recall which file (order file? mos file? or model itself?)